cluster/addons/registry/images/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM haproxy:1.5
```

Comment:
This feels like a pretty heavy base image when all we need is proper forwarding.  But as discussed, the registry backend was not working properly when simply forwarded with socat